lib/entities/integrations/index.ts

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+import {buildApiUrl, NetworkClient} from "../../network";
+import {IIntegration} from "./models";
+import {wrapWithErrorHandler} from "../../network/utils";
+
+export class Integrations {
+    networkClient: NetworkClient;
+    baseUrl: string = "integrations";
+
+    constructor(networkClient: NetworkClient) {
+        this.networkClient = networkClient;
+    }
+
+    /**
+     * Retrieve all integrations that matches the specified plan identifier
+     * @param planIdentifier - Identifier of the plan that the integration belongs to.
+     * @returns All integrations on the plan including its relevant information about the app
+     */
+    public async getAll(planIdentifier:string): Promise<IIntegration[] | undefined> {
```

Comment:
Spacing needed here for `planIdentifier:string` so it should be `planIdentifier: string` per best practise :)